governance/program/src/instruction.rs

```diff
@@ -378,10 +378,10 @@ pub enum GovernanceInstruction {
         config: GovernanceConfig,
 
         #[allow(dead_code)]
-        /// Indicates whether token owner should be transferred to the Governance PDA
+        /// Indicates whether the token account authorities (AccountOwner and optionally CloseAccount) should be transferred to the Governance PDA
```

Comment:
The `CloseAccount` authority is rarely the owner, so I'm not sure of the use-case for this

Reply:
In that case, I would just check that it's `None` and that's it. If it's `None` then the owner is the CloseAuthority by default